README.md

```diff
@@ -90,6 +88,44 @@ final class LastPosts extends Widget
 }
 ```
 
+### LocaleState of `View` and `WebView` services
```

Comment:
```suggestion
#### Language in state
```

This is subheading of "State of `View` and `WebView` services" part.

Reply:
It's locale, not language.